src/wmnds/components/content-tiles/page-contents/page-contents.njk

```diff
@@ -0,0 +1,21 @@
+{% macro wmndsContentTilePageContents(params) %}
+
+<div class="wmnds-content-tile wmnds-content-tile--page-contents">
+    <div class="wmnds-content-tile__title">
+    <h2>Page Contents</h2>
```

Comment:
Similar to the other issue, this h2 should have the title class and then we can get rid of the div
```suggestion
    <h2 class="wmnds-content-tile__title">Page Contents</h2>
```